Azure/azure-cli
# [AppService] `az functionapp create`: Create consumption plan before creating function app

**Related command**
`az functionapp create`

**Description**
Resolves part 1 of #27533 

**Testing Guide**
Create consumption function app and notice that the naming convention for the plan is different. 

---

This checklist is used to make sure that common guidelines for a pull request are followed.
